tests/bdd/steps/the_response.py

```diff
@@ -52,7 +53,7 @@ def the_response_body_matches(context, regex):
 
 
 @then("the response body does not match '{regex}'")
-def the_response_body_matches(context, regex):
+def the_response_body_does_not_match(context, regex):
```

Comment:
How on earth did this come up in linting? Or did you just spot this yourself?

Reply:
Linting noticed that two test methods has the exact same name. What that was doing to the tests, I don't know